Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.1.0"
 
 [deps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Pluto = "c3e4b0f8-55cb-11ea-2926-15256bba5781"
```

Comment:
Is there a need for Pluto to be added as a dependency? The notebooks are independent.

Reply:
Seems it is not needed since Pluto is portable